src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -162,12 +187,17 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <ProduceReferenceAssembly Condition="'$(ProduceReferenceAssembly)' == ''">false</ProduceReferenceAssembly>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(OutputPath)' == '' ">
+  <!--
+    For non-Sdk style projects, the validation of an invalid Configuration/Platform is determined by the non-empty
```

Comment:
Sure. I recommend that we should not use `.NET Core`, as it is being rebranded into `.NET`. Also, I think, it would be better to say the actual project system used instead of specifically mentioning `.NET SDK`. Since, CPS (_is not tied to `.NET` only, right?!_) might replace old project system altogether by supporting the rest of the project system flavours.

```suggestion
    For projects loaded with MPFProj (Legacy project system), the validation of an invalid Configuration/Platform is determined by the non-empty
```